documentation/dependencies/debian.dependencies

```diff
@@ -26,3 +26,4 @@ sqlite3
 pv
 rsync
 ccache
+libguestfs-tools
```

Comment:
Nice catch!
I think we are missing libguestfs on archlinux dependencies file as well.

Reply:
Added for archlinux :)